### PR TITLE
ssh_username terraform variable

### LINKED ISSUE #349: Use core as SSH user for CoreOS Container Linux
**What happened**:
Used "coreos-stable" as image.

**What is the expected behavior**:
I expect that the output for `tf.json` uses `core` as ssh_user.

**How to reproduce the issue**:
Set `droplet_image = "coreos-stable"` in `terraform.tfvars` before running `terraform apply`.
Then with `terraform output -json > tf.json` you get the output with:

```json
    ...
    "kubeone_hosts": {
        "sensitive": false,
        "type": "map",
        "value": {
            "control_plane": [
                {
                    "cloud_provider": "digitalocean",
                    "cluster_name": "kubeone",
                    "private_address": [
                        "10.10.25.25",
                        "10.10.25.26",
                        "10.10.25.27"
                    ],
                    "public_address": [
                        "142.93.142.142",
                        "142.93.142.143",
                        "142.93.142.144"
                    ],
                    "ssh_agent_socket": "env:SSH_AUTH_SOCK",
                    "ssh_port": "22",
                    "ssh_private_key_file": "",
                    "ssh_user": "root"
                }
            ]
        }
    },
```

**Anything else we need to know?**

**Information about the environment**:
KubeOne version (`kubeone version`): master
Operating system: Arch Linux
Provider you're deploying cluster on: DigitalOcean
Operating system you're deploying on: CoreOS Container Linux

### PR DESCRIPTION
**Pull request by kron4eg**
Fixes #349 

```release-note
ssh_username variable to be used in output in terraform examples
```

### CODE REVIEW

File: examples/terraform/digitalocean/variables.tf [hunk 1]
```diff
@@ -38,6 +38,11 @@ variable "ssh_port" {
   description = "SSH port to be used to provision instances"
 }
 
+variable "ssh_user" {
+  default = "root"
+  description = "SSH user, used only in output"
```
Comment (xmudrii) [inline examples/terraform/digitalocean/variables.tf]: It isn't only used in the output, but it's also used by KubeOne to connect to the instances. Maybe we should improve the description to mention that.

Comment (kron4eg) [reply examples/terraform/digitalocean/variables.tf]: yes, that's what outputs are used

File: examples/terraform/aws/output.tf [hunk 2]
```diff
@@ -30,7 +30,7 @@ output "kubeone_hosts" {
       ssh_agent_socket     = "${var.ssh_agent_socket}"
       ssh_port             = "${var.ssh_port}"
       ssh_private_key_file = "${var.ssh_private_key_file}"
-      ssh_user             = "ubuntu"
+      ssh_user             = "${var.ssh_username}"
```
Comment (xmudrii) [inline examples/terraform/aws/output.tf]: Why the variable name is not the same as in the output?

Comment (kron4eg) [reply examples/terraform/aws/output.tf]: for historical reasons, `ssh_user` is kubeone's API, `$var.ssh_username` is for consistency (GCE example already had this).

**Comment by kron4eg** [comment]
/override pull-kubeone-e2e-digitalocean-conformance-1.13.5
/override pull-kubeone-e2e-digitalocean-upgrade-1.13.5-1.14.1

DO API yet again doesn't feel good

**Comment by metalmatze** [comment]
Thanks! :clap:

**Pull request merged**